guard.go

```diff
@@ -0,0 +1,18 @@
+package rocket
+
+import (
+	"net/http"
+)
+
+type Action uint8
+
+const (
+	_ Action = iota
+	Success
+	Failure
+	Forward
```

Comment:
can you explain what is `Forward` doing?
In my opinion, the `VerifyRequest` only need to return the error to represent the verify is passed or not.

Reply:
It means we neither success nor failure, I'm thinking do we need this since we didn't early return with success.

Reply:
can you explain more specific?
In the current implementation, I think the error is the only thing we care, aren't we?

Reply:
Yes, I think too less on this implementation, for now, I think we just need to return error